test/korma_encrypted/core_test.clj

```diff
@@ -46,14 +46,15 @@
     (sql/db-do-commands
        spec
        (tap (sql/create-table-ddl "credit_cards"
-                                  [:id "serial primary key"]
```

Comment:
Definitely a feature we want to add, at least down the line. We opted for pragmatism for use in our current apps in this case over spending the time to do that.  If you want to pick that up, that'd be awesome.